doc/minversions.md

```diff
@@ -18,14 +18,24 @@ myst:
 # Minimum Supported Versions
 
 PyGMT has adopted [SPEC 0](https://scientific-python.org/specs/spec-0000/) alongside the
-rest of the scientific Python ecosystem, and therefore:
+rest of the scientific Python ecosystem, and will therefore:
 
-- Support for Python versions be dropped 3 years after their initial release.
-- Support for core package dependencies (NumPy, pandas, Xarray) be dropped 2 years after
-  their initial release.
+- Drop support for Python versions 3 years after their initial release.
+- Drop support for core package dependencies (NumPy, pandas, Xarray) 2 years after their
+  initial release.
 
-Similarly, the PyGMT team has decided to discontinue support for GMT versions 3 years
-after their initial release.
+In addition to the above, the PyGMT team has also decided to:
+
+- Drop support for GMT versions 3 years after their initial release, while ensuring at
+  least two latest minor versions remain supported.
+- Maintain support for optional dependencies for at least 1 year after their initial
+  release. Users are encouraged to use the most up-to-date optional dependencies where
+  possible.
```

Comment:
Maybe link to https://pygmt-dev--3616.org.readthedocs.build/en/3616/ecosystem.html#pygmt-dependencies so people know what the optional dependencies are.

```suggestion
- Maintain support for [optional dependencies](/ecosystem.md#pygmt-dependencies) for at
  least 1 year after their initial release. Users are encouraged to use the most
  up-to-date optional dependencies where possible.
```